openlcsd/flow/tasks.py

```diff
@@ -783,17 +854,13 @@ def fork_component_imports(context, engine):
             ),
             get_container_components,
             save_container_components,
-            # Get the source path for each component from container source
-            # For components failed to get a source path, exception warning
-            # should be logged.
-            get_component_source_path,
             # Fork the import task for each component
-            fork_component_imports
+            fork_components_imports,
         ],
 
         # Task flow for other build types
         [
-            download_package_archive,
+            download_component,
```

Comment:
use `download_source`?

Reply:
Updated to download_component_source, is that acceptable?